src/Map.mo

```diff
@@ -1183,13 +1183,13 @@ module {
   ///
   /// Note: Creates `O(log(n))` temporary objects that will be collected as garbage.
   public func toText<K, V>(map : Map<K, V>, keyFormat : K -> Text, valueFormat : V -> Text) : Text {
-    var text = "";
+    var text = "{";
```

Comment:
(See above comment)

```suggestion
    if (size(map) == 0) {
      return "{}"
    };
    var text = "{ ";
```

Reply:
I believe so, since it's consistent with Motoko's code style. 

Since the `toText()` format is highly subjective, a resemblance to the Motoko language syntax is probably the least opinionated choice (essentially a [focal point](https://en.wikipedia.org/wiki/Focal_point_(game_theory))).